src/ucp/wireup/wireup.h

```diff
@@ -13,6 +13,10 @@
 #include <uct/api/uct.h>
 #include <ucs/arch/bitops.h>
 
+typedef struct {
+    uint64_t mandatory;
+    uint64_t optional;
```

Comment:
1. 2 space lines fore struct
2. document the struct and its fields. 
"optional" - in addition to all mandatory flags, at least one of the flags defined by it must be present